signingvalidation.proj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <Target Name="Build"
+          DependsOnTargets="BuildPrerequisites;
+                            Validate" />
+
+  <Target Name="BuildPrerequisites"
+          DependsOnTargets="CreateVersionInfoFile;
+                            BuildCustomTasks" />
+
+  <Target Name="Validate">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)tools-local\signingvalidation\dir.proj" />
```

Comment:
I want the signing validation to be a self-contained thing, so that in our build definition, we can just build one project and it does all the things necessary for it to run, including building its prerequisites (local MSBuild tasks, creating the buildversion.props file).  This is preferable to knowing that in order to do signing validation, I need to build the local tasks, create the version info file, and then build the validation project. My testing showed that I couldn't get MSBuild to re-evaluate its properties if I called MSBuild on the same project file.  So, I was forced to create a separate project file which I can MSBuild.